package.json

```diff
@@ -2,14 +2,12 @@
   "name": "oam-uploader-api",
   "version": "1.0.0",
   "description": "An uploader API for Open Aerial Map Imagery",
-  "main": "index.js",
   "scripts": {
     "docker-build": "docker build -t oam-uploader-api .build_scripts/docker",
     "docker-install": ".build_scripts/docker/run.sh /install.sh",
     "docker-test": ".build_scripts/docker/run.sh /test.sh",
     "docker-start": ".build_scripts/docker/run.sh /start.sh",
     "docker-console": ".build_scripts/docker/run.sh bash",
-    "start": "TMPDIR=/tmp node index.js",
```

Comment:
I couldn't find the reason it was added; that's the default for `TMPDIR` (and if it's overridden by the system, it's usually for good reason).